src/Administration/KnownIps/KnownIpsPageLoader.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Administration\KnownIps;
+
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class KnownIpsPageLoader
```

Comment:
Why not just an service which collects them?

Reply:
You are right. Page loading is not a common pattern for the administration.